<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CKJVTextFormatConfig</class>
 <widget class="QWidget" name="CKJVTextFormatConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Format Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="3">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="vertLayoutColorOptions">
       <item>
        <widget class="QLabel" name="labelColorOptions">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Color Settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetColorOptions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonWordsOfJesusColor">
         <property name="text">
          <string>Words of Jesus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSearchResultsColor">
         <property name="text">
          <string>Search Results</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCursorFollowColor">
         <property name="text">
          <string>Cursor Tracker</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_2">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelScriptureBrowserFont">
       <property name="text">
        <string>Scripture &amp;Browser Font:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>fontComboBoxScriptureBrowser</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFontComboBox" name="fontComboBoxScriptureBrowser"/>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="dblSpinBoxScriptureBrowserFontSize"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSearchResultsFont">
       <property name="text">
        <string>Search &amp;Results Font:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>fontComboBoxSearchResults</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFontComboBox" name="fontComboBoxSearchResults"/>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="dblSpinBoxSearchResultsFontSize"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxInvertTextBrightness">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Invert Text Brightness:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horzSliderTextBrigtness">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxAdjustDialogElementBrightness">
       <property name="text">
        <string>Adjust &amp;Dialog Element Brightness</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="treeViewSearchResultsPreview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>10</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>320</height>
         </size>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textScriptureBrowserPreview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>20</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>320</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fontComboBoxScriptureBrowser</tabstop>
  <tabstop>dblSpinBoxScriptureBrowserFontSize</tabstop>
  <tabstop>fontComboBoxSearchResults</tabstop>
  <tabstop>dblSpinBoxSearchResultsFontSize</tabstop>
  <tabstop>checkBoxInvertTextBrightness</tabstop>
  <tabstop>horzSliderTextBrigtness</tabstop>
  <tabstop>checkBoxAdjustDialogElementBrightness</tabstop>
  <tabstop>listWidgetColorOptions</tabstop>
  <tabstop>buttonWordsOfJesusColor</tabstop>
  <tabstop>buttonSearchResultsColor</tabstop>
  <tabstop>buttonCursorFollowColor</tabstop>
  <tabstop>treeViewSearchResultsPreview</tabstop>
  <tabstop>textScriptureBrowserPreview</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
